rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -29,569 +29,550 @@
 import rocal_pybind as b
 from amd.rocal.pipeline import Pipeline
 
-def blend(*inputs,ratio=None):
-    kwargs_pybind = {"input_image0":inputs[0], "input_image1":inputs[1], "is_output":False ,"ratio":ratio}
-    blend_image = b.Blend(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+
+def blend(*inputs, ratio=None, rocal_tensor_layout=types.NHWC,
+          rocal_tensor_output_type=types.UINT8):
+    ratio = b.createFloatParameter(
+        ratio) if isinstance(ratio, float) else ratio
+    # pybind call arguments
+    kwargs_pybind = {"input_image0": inputs[0], "input_image1": inputs[1], "is_output": False, "ratio": ratio,
+                     "rocal_tensor_output_layout": rocal_tensor_layout, "rocal_tensor_output_datatype": rocal_tensor_output_type}
+    blend_image = b.Blend(
+        Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (blend_image)
 
-def snow(*inputs, snow=0.5, device=None):
+
+def snow(*inputs, snow=0.5, rocal_tensor_layout=types.NHWC,
+         rocal_tensor_output_type=types.UINT8):
+    snow = b.createFloatParameter(snow) if isinstance(snow, float) else snow
     # pybind call arguments
-    snow = b.CreateFloatParameter(snow) if isinstance(snow, float) else snow
-    kwargs_pybind = {"input_image0": inputs[0],"is_output": False, "shift": snow}
-    snow_image = b.Snow(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "shift": snow,
+                     "rocal_tensor_output_layout": rocal_tensor_layout, "rocal_tensor_output_datatype": rocal_tensor_output_type}
+    snow_image = b.Snow(Pipeline._current_pipeline._handle,
+                        *(kwargs_pybind.values()))
     return (snow_image)
 
-def exposure(*inputs, exposure=0.5, device=None):
+
+def exposure(*inputs, exposure=0.5, rocal_tensor_layout=types.NHWC,
+             rocal_tensor_output_type=types.UINT8):
+    exposure = b.createFloatParameter(
+        exposure) if isinstance(exposure, float) else exposure
     # pybind call arguments
-    exposure = b.CreateFloatParameter(exposure) if isinstance(exposure, float) else exposure
-    kwargs_pybind = {"input_image0": inputs[0],"is_output": False, "shift": exposure}
-    exposure_image = b.Exposure(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "shift": exposure,
+                     "rocal_tensor_output_layout": rocal_tensor_layout, "rocal_tensor_output_datatype": rocal_tensor_output_type}
+    exposure_image = b.Exposure(
+        Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (exposure_image)
 
-def fish_eye(*inputs, device=None):
+
+def fish_eye(*inputs, rocal_tensor_layout=types.NHWC,
+             rocal_tensor_output_type=types.UINT8):
```

Comment:
Follow the comment for entire file

Reply:
I formatted the file using autopep8 tool. We need to commit to a formatting tool for our python files and this is the default python formatter